org.mythtv.MythTV.yaml

```diff
@@ -0,0 +1,1063 @@
+#        tag: rel/1.10.15
+#        commit: 92fe5ca29625d30b98704806d6379fe9acb001e3
+#    build-options:
+#      env:
+#        JAVA_HOME: 
+
+  - name: p8-platform
+    sources:
+      - type: git
+        url: https://github.com/Pulse-Eight/platform.git
+        commit: 2748be52ae27e6007ef548b697d4a03ff7de4291
+    buildsystem: cmake-ninja
+
+  - name: libcec
+    sources:
+      - type: git
+        url: https://github.com/Pulse-Eight/libcec.git
+        tag: libcec-7.0.0
+        commit: ceeec8bfd2242792b6de59ce8fb854437208dc11
+    buildsystem: cmake-ninja
+
+  - name: libhdhomerun
+    sources:
+      - type: git
+        url: https://github.com/Silicondust/libhdhomerun.git
+        commit: e8f29fd4e071580ffb583cbeb9cf720c96a4e75f
+    buildsystem: simple
+    # no make install in makefile
+    build-commands:
+      - make && make install
+      - mkdir -p /app/include/libhdhomerun
+      - cp *.h /app/include/libhdhomerun
+      - cp libhdhomerun.so /app/lib
+
+  - name: libass
+    sources:
+      - type: git
+        url: https://github.com/libass/libass.git
+        tag: 0.17.3
+        commit: e46aedea0a0d17da4c4ef49d84b94a7994664ab5
+
+#  - name: mmal
+#    sources:
+#      - type: git
+#        url: https://github.com/raspberrypi/userland.git
+#        commit: a54a0dbb2b8dcf9bafdddfc9a9374fb51d97e976
+#    buildsystem: cmake
+#    subdir: interface/mmal
+
+#  - name: libiec61883
+#    sources:
+#      - type: git
+#        url: https://git.kernel.org/pub/scm/libs/ieee1394/libiec61883.git/
+#        tag: v1.2.0
+#        commit: 6a7c0bd85c0f4ac2861f96547ab11d0d469f38ca
+
+#  - name: dveo-asi
+#    sources:
+#      - type: git
+#        url: https://github.com/kierank/dveo-linux-master.git
+#        commit: 5d62458bb6e13822f59f85eb260e31bb2c7df3a2
+
+#MythBrowser requires QtWebkit.
+#MythNetvision requires the MythTV Python bindings (MythTV)
+#MythNetvision requires the Python pycurl library (pycurl)
+#Disabling MythNetvision due to missing dependencies.
+#MythWeather requires Date::Manip
+#MythWeather requires XML::XPath
+#MythWeather requires Image::Size
+#MythWeather requires DateTime::Format::ISO8601
+#MythWeather requires SOAP::Lite
+#MythWeather requires JSON
+#Disabling MythWeather due to missing dependencies.
+#MusicBrainz support requires libdiscid.
+#Disabling MythZMServer. Requires mysql/mysql.h header
+# libcdio        support will not be included in MythMusic
+
+  - name: mythtv
+    sources:
+      - type: git
+        url: https://github.com/MythTV/mythtv.git
+        branch: fixes/35
+        commit: 30ba688773852d3f8ad44dd7ceb1b6634ca7ea4e
+      - type: file
+        path: mythtv128x128.png
+      - type: file
+        path: org.mythtv.MythTV.desktop
+      - type: file
+        path: org.mythtv.MythTV.metainfo.xml
+      - type: patch
+        path: libva.patch
+    buildsystem: simple
+    build-options:
+      prepend-pkg-config-path: /app/lib64/pkgconfig
+      ldflags: -L/app/lib64
```

Comment:
lib64 is not supposed to exist it should be /app/lib, you were using an old flatpak-builder version.